packages/datetime/src/datePicker.tsx

```diff
@@ -7,7 +7,9 @@
 import { AbstractPureComponent, Button, IProps, Utils } from "@blueprintjs/core";
 import * as classNames from "classnames";
 import * as React from "react";
-import * as ReactDayPicker from "react-day-picker";
+import { default as ReactDayPicker } from "react-day-picker";
```

Comment:
Whoa, funky syntax. Haven't seen this before, but it's interesting. If it's equivalent, should we just do:
```tsx
import ReactDayPicker from "react-day-picker";
```
?

_Source: https://stackoverflow.com/a/31092287/5199574_

Reply:
yes

Reply:
It would have to be `import DayPicker from "react-day-picker";`, since the default export is `DayPicker` (not `ReactDayPicker`). However, that conflicts with the actual `DayPicker` component name. So used this approach to rename `DayPicker` to `ReactDayPicker`. Open to other alternatives, but as far as I know, the above recommendation won't work?

Reply:
> It would have to be import DayPicker from "react-day-picker";, since the default export is DayPicker (not ReactDayPicker)

Are you sure about this? `default` exports are (by design) anonymous, so you can import them with whatever name you want.

Reply:
@adidahiya will that work with the defined types though? 

https://github.com/gpbl/react-day-picker/blob/3156e3de1664caa15cf12cfd0fdd96350874e8c5/types/index.d.ts#L4

Reply:
it will. default exports don't have a name of their own. you can name them whatever you want with the `import DefaultName from "package"` syntax.

Reply:
Wow. I swear I tried that 🤔 TIL. Appreciate the feedback! 😊